muspinsim_plot/muspinsim_plot.xml

```diff
@@ -98,7 +102,11 @@
             <param name="linestyle" value="solid"/>
             <param name="colour" value="b"/>
             <param name="linewidth" value="1.0"/>
-            <output name="out_file_pdf" ftype="pdf" file="fitting_example.pdf" compare="sim_size"/>
+            <output name="out_file_pdf" ftype="pdf">
+                <assert_contents>
+                    <has_size value="14785"/>
```

Comment:
All the tests pass on my local machine except for this one where the file size in marginally different 
`Expected file size of 14785+-0 found 14791`

```suggestion
                    <has_size value="14785" delta="20"/>
```